<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ToolDialog</class>
 <widget class="QDialog" name="ToolDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>559</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Tool</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="lblName">
     <property name="text">
      <string>Na&amp;me:</string>
     </property>
     <property name="buddy">
      <cstring>edtName</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLineEdit" name="edtName">
       <property name="whatsThis">
        <string>The name will be displayed in the 'Tools-&gt;External Tools' menu.</string>
       </property>
       <property name="placeholderText">
        <string>Short name of the tool</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KIconButton" name="btnIcon"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblExecutable">
     <property name="text">
      <string>Executable:</string>
     </property>
     <property name="buddy">
      <cstring>edtExecutable</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="KUrlRequester" name="edtExecutable">
     <property name="placeholderText">
      <string>Application or interpreter</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lblArgs">
     <property name="text">
      <string>Ar&amp;guments:</string>
     </property>
     <property name="buddy">
      <cstring>edtArgs</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="edtArgs">
     <property name="placeholderText">
      <string>Command line arguments</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lblInput">
     <property name="text">
      <string>&amp;Input:</string>
     </property>
     <property name="buddy">
      <cstring>edtInput</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QTextEdit" name="edtInput">
     <property name="placeholderText">
      <string>Optional standard input</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="lblWorkingDir">
     <property name="text">
      <string>Working directory:</string>
     </property>
     <property name="buddy">
      <cstring>edtWorkingDir</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="KUrlRequester" name="edtWorkingDir">
     <property name="placeholderText">
      <string>Uses current document path if empty</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="lblMimeType">
     <property name="text">
      <string>Mime &amp;types:</string>
     </property>
     <property name="buddy">
      <cstring>edtMimeType</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="edtMimeType">
       <property name="whatsThis">
        <string>A semicolon-separated list of mime types for which this tool should be available. If empty, the tool is always available.</string>
       </property>
       <property name="placeholderText">
        <string>Show tool only for given mime types</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnMimeType">
       <property name="whatsThis">
        <string>Opens a dialog that helps in creating a list of mimetypes.</string>
       </property>
       <property name="icon">
        <iconset theme="tools-wizard"/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="lblSave">
     <property name="text">
      <string>Save:</string>
     </property>
     <property name="buddy">
      <cstring>cmbSave</cstring>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QComboBox" name="cmbSave">
     <property name="whatsThis">
      <string>Optionally save the current or all modified documents prior to running the command. This is helpful if you want to pass URLs to an application like, for example, an FTP client.</string>
     </property>
     <item>
      <property name="text">
       <string>None</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Current Document</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>All Documents</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Trigger</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QComboBox" name="cmbTrigger">
     <item>
      <property name="text">
       <string>None</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Before Save</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>After Save</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QCheckBox" name="chkReload">
     <property name="text">
      <string>Reload current document after execution</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>O&amp;utput:</string>
     </property>
     <property name="buddy">
      <cstring>cmbOutput</cstring>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QComboBox" name="cmbOutput">
     <item>
      <property name="text">
       <string>Ignore</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Insert at Cursor Position</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Replace Selected Text</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Replace Current Document</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Append to Current Document</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Insert in New Document</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Copy to Clipboard</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Display in Pane</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Editor command:</string>
     </property>
     <property name="buddy">
      <cstring>edtCommand</cstring>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="QLineEdit" name="edtCommand">
     <property name="whatsThis">
      <string>If you specify a name here, you can invoke the command from the view command line with the_name_you_specified_here. Please do not use spaces or tabs in the name.</string>
     </property>
     <property name="placeholderText">
      <string>Optional view command line name</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KIconButton</class>
   <extends>QPushButton</extends>
   <header>kiconbutton.h</header>
  </customwidget>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QWidget</extends>
   <header>kurlrequester.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>edtName</tabstop>
  <tabstop>btnIcon</tabstop>
  <tabstop>edtExecutable</tabstop>
  <tabstop>edtArgs</tabstop>
  <tabstop>edtInput</tabstop>
  <tabstop>edtWorkingDir</tabstop>
  <tabstop>edtMimeType</tabstop>
  <tabstop>btnMimeType</tabstop>
  <tabstop>cmbSave</tabstop>
  <tabstop>chkReload</tabstop>
  <tabstop>cmbOutput</tabstop>
  <tabstop>edtCommand</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
